### PR TITLE
[ADD] purchase_exception: migration script to v12

### PR DESCRIPTION
**Pull request by MiquelRForgeFlow**
Add migration script.

### CODE REVIEW

**Comment by MiquelRForgeFlow** [comment]
@sudhir-erpharbor

File: purchase_exception/migrations/12.0.1.0.1/noupdate_changes.xml [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+  <record id="ir_cron_test_po_order_except" model="ir.cron" forcecreate="True">
+    <field name="code">model.test_all_draft_orders()</field>
+  </record>
+  <record id="po_excep_no_email" model="exception.rule">
+<!--    <field name="rule_group"/>-->
+    <field name="code">if not self.partner_id.email:
+            failed=True</field>
+  </record>
+  <record id="pol_excep_qty_check" model="exception.rule">
+<!--    <field name="rule_group"/>-->
```
Comment (HviorForgeFlow) [inline purchase_exception/migrations/12.0.1.0.1/noupdate_changes.xml:12]: Cleanup unused code

**Review by HviorForgeFlow — COMMENTED** [review]
Can you provide a little explanation about what this migration script is fixing? Thanks

File: purchase_exception/migrations/12.0.1.0.1/noupdate_changes.xml (hunk 1 above)
Comment (MiquelRForgeFlow) [reply purchase_exception/migrations/12.0.1.0.1/noupdate_changes.xml:12]: It's better to not delete in this case. If in the future we check again the noupdate changes from v11 and v12 versions, then this way we will not be tempted to add the commented code.

**Comment by MiquelRForgeFlow** [comment]
@hveficent In v12, respect to v11, some changes have been done in xml records that are with "noupdate = 1". So, in order to make effective this changes during migration, this code is needed.

**Review by HviorForgeFlow — APPROVED** [review]

**Review by AaronHForgeFlow — APPROVED** [review]
:+1: LGTM

**Comment by HviorForgeFlow** [comment]
@pedrobaeza I'm not sure if ocabot can merge this fix to migration script from 12.0.1.0.1 because it will change version. What should we do?

**Comment by pedrobaeza** [comment]
we can merge without version bump:

/ocabot merge

et voilá

**Comment by HviorForgeFlow** [comment]
I didin't realize about that way :smile: 
Thanks you so much!

**Pull request merged**